ipamapi/contract.go

```diff
@@ -42,59 +51,22 @@ type AddressSpace string
 // Common key is a addressspace
 type Config interface {
```

Comment:
Yes. I understand the original intention behind these APIs. But, having these APIs exposed prevent the applications/drivers from using the Config APis in order to make use of the Request API. I think these Config APIs must be driver specific & not to be prescribing by the core. WDYT ?